content/ui-patterns/overlays.mdx

```diff
@@ -0,0 +1,415 @@
+    alt=" "
+    style="background: none"
+  />
+</ImageContainer>
+<Caption>
+
+In order to work well in non-rectangular mobile screens and browser UI, bottom sheets may add extra padding inside the overlay, following [`env(safe-area-inset-*)`](https://developer.mozilla.org/en-US/docs/Web/CSS/env()) environment variables.
+
+</Caption>
+
+On landscape, a bottom sheet has a maximum width of 480px, centered horizontally.
+
+<ImageContainer>
+    <img
+        width="880"
+        src="https://user-images.githubusercontent.com/293280/147170826-73caa091-b71a-4296-beba-ccf3a2ccf9ab.png"
+        alt=" "
+        style="background: none"
+    />
+</ImageContainer>
+<Caption>Bottom sheet on landscape viewports.</Caption>
+
+<ImageContainer>
+    <img
+        width="500"
+        src="https://user-images.githubusercontent.com/293280/147170671-09e6002a-2622-41b6-afe5-f37f0890a6c3.png"
+        alt=" "
+        style="background: none"
+    />
+</ImageContainer>
+<Caption>Header indicator when no title and close button are present.</Caption>
+
+A bottom sheet without a title or close button gets an a header indicator at the top. Slide it down to dismiss the overlay.
+
+### Full-screen (specific to narrow viewports)
+
+Full-screen overlays may be used to present content that needs all the available space on narrow viewports.
+
+Overlays that have one or more input fields should always use the full-screen variant.
+
+<!--
+## Navigation
+This section should cover in-overlay navigation such as drill-in scenarios, back button, etc.
+-->
+
+## Components
+
+Several components that rely on overlays are available for use. Use them directly when aiming for a specific behavior:
+
+<Grid gridTemplateColumns={['1fr', null, null, null, '1fr 2fr']} gridGap={4}>
+    <div>
+        <Link href="https://primer.style/react/ActionMenu">
+            <img alt="Action menu" src="https://user-images.githubusercontent.com/293280/123880964-95b3a700-d8f8-11eb-9775-cbaf165207ed.png" />
+        </Link>
+    </div>
+    <div>
+        <Heading fontSize={3}>Action menu</Heading>
+        <Box as="p">
+
+Action menus are used for disambiguation, navigation, or to display secondary options. They appear when users interact with buttons, actions, or other controls.
+
+[Primer React implementation](https://primer.style/react/ActionMenu)
+
+</Box>
+    </div>
+    <div>
+        <Link href="https://primer.style/react/DropdownMenu">
+            <img alt="Dropdown menu" src="https://user-images.githubusercontent.com/293280/123881067-cdbaea00-d8f8-11eb-98e4-e57c64489308.png" />
+        </Link>
+    </div>
+    <div>
+        <Heading fontSize={3}>Dropdown menu</Heading>
+        <Box as="p">
+Dropdown menus are used for making a single selection among a small list of options. The list is usually predefined with system values, but in some cases it can include user-provided data when those are known not to grow into too many items.
```

Comment:
```suggestion

Dropdown menus are used for making a single selection among a small list of options. The list is usually predefined with system values, but in some cases it can include user-provided data when those are known not to grow into too many items.
```

Adding an extra line above the para to see if the link spacing gets fixed.

Reply:
> but in some cases it can include user-provided data when those are known not to grow into too many items.

This is confusing to me.